src/ResourceManager/ServiceBus/Commands.ServiceBus/Models/ServiceBusDRConfigurationAttributes.cs

```diff
@@ -0,0 +1,64 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.ServiceBus.Models;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.ServiceBus.Models
+{
+
+    public class ServiceBusDRConfigurationAttributes
```

Comment:
@v-Ajnava this is optional, but I would suggest create `PS` wrapper classes for any SDK model object you plan to expose in your cmdlets (be it as output or parameter types). For example, this class would look like the following:

```cs
public class PSArmDisasterRecovery : ArmDisasterRecovery
{
    // Add additional properties you may need that aren't found in SDK type
}
```

This makes the class simpler and follows other module's pattern of having `PS` objects. Let me know what you think.